tests/unit/test_utils.py

```
@@ -0,0 +1,47 @@
+
+        # Block string
+        query_block = 'field(arg: """block # string""")'
+        stripped_block = _strip_comments(query_block)
+        self.assertIn('""""""', stripped_block)  # It replaces block strings
+
+    def test_format_kb(self):
+        """
+        Verify the logic of the real format_kb function directly.
+        """
+        # Test overflow/special cases
+        self.assertEqual(format_kb(float("inf")), "inf")
```

Ruff (0.14.14)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)